remote.go

```diff
@@ -54,6 +56,15 @@ func parseRepoInfo(remote string, docker bool) (*RepoInfo, error) {
 		r.Auth = "https://auth.docker.io/token"
 		r.Service = "registry.docker.io"
 		r.Docker = true
+	} else if r.Host == "iad.ocir.io" {
+		r.Auth = "https://iad.ocir.io/20180419/docker/token"
+		r.Service = "iad.ocir.io"
+		r.Docker = true
+		if data.User == nil {
+
+			r.Username = os.Getenv("IAD_USERNAME")
```

Comment:
If you want to use env vars to pass these on then I think they should just be "DOCKER_USERNAME" "DOCKER_PASSWORD"  and you should set them further down  

having the ability to not require passwords in the URL would be useful , especially for pulling but I don't think it's Oracle specific.